internal/provisioner/helm/controllers/bundledeployment_controller.go

```diff
@@ -340,6 +383,54 @@ func (r *BundleDeploymentReconciler) loadChart(ctx context.Context, bundle *rukp
 	return getChart(chartfs)
 }
 
+func (r *BundleDeploymentReconciler) addDynamicWatch(cl helmclient.ActionInterface, bd *rukpakv1alpha1.BundleDeployment, bundle *rukpakv1alpha1.Bundle) error {
+	release, err := cl.Get(bd.GetName())
+	if err != nil {
+		return err
+	}
+	var objects []client.Object
+	dec := apimachyaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(release.Manifest)), 1024)
+	for {
+		obj := unstructured.Unstructured{}
+		err := dec.Decode(&obj)
+		if errors.Is(err, io.EOF) {
+			break
+		}
+		if err != nil {
+			return fmt.Errorf("read %q: %v", obj, err)
```

Comment:
Is printing the obj here potentially problem for logs? Or does this just print the metadata information we care about?

Reply:
The `err` has enough information.  I take out the `obj`.